content/pt-br/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -0,0 +1,219 @@
+        }
+    }
+}
+```
+
+{{< note >}}
+Se você usar o armazenador de credenciais do Docker, você não vai ver essa entrada `auth`, mas sim uma `credsStore` com o nome do armazenador como valor.
+{{< /note >}}
+
+## Crie um segredo com base nas credenciais existentes {#registry-secret-existing-credentials}
+
+Um cluster de Kubernetes usa o Segredo do tipo `kubernetes.io/dockerconfigjson` para autenticar com um registro de contêineres
+para pegar uma imagem privada.
+
+Se você já executou `docker login`, você pode copiar
+essa credencial para o Kubernetes:
+
+```shell
+kubectl create secret generic regcred \
+    --from-file=.dockerconfigjson=<path/to/.docker/config.json> \
+    --type=kubernetes.io/dockerconfigjson
+```
+
+Se você precisar de mais controle (por exemplo, para definir um namespace ou um novo
+segredo) então você pode personalizar o segredo antes de armazená-lo.
+Tenha certeza de:
+
+- definir o nome do item de dados como `.dockerconfigjson`
+- codificar com base64 o arquivo de configuração do Docker e então colar essa String, inteira, 
+como o valor para o campo `data[".dockerconfigjson"]`
+- definir o `type` para `kubernetes.io/dockerconfigjson`
+
+Exemplo:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: myregistrykey
+  namespace: awesomeapps
+data:
+  .dockerconfigjson: UmVhbGx5IHJlYWxseSByZWVlZWVlZWVlZWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWxsbGxsbGxsbGxsbGxsbGxsbGxsbGxsbGxsbGxsbGx5eXl5eXl5eXl5eXl5eXl5eXl5eSBsbGxsbGxsbGxsbGxsbG9vb29vb29vb29vb29vb29vb29vb29vb29vb25ubm5ubm5ubm5ubm5ubm5ubm5ubm5ubmdnZ2dnZ2dnZ2dnZ2dnZ2dnZ2cgYXV0aCBrZXlzCg==
+type: kubernetes.io/dockerconfigjson
+```
+
+Se você receber a mensagem de erro `error: no objects passed to create`, isso pode significar que a sequência codificada com base64 é inválida.
+Se você receber uma mensagem de erro como `Secret "myregistrykey" é inválida: data[.dockerconfigjson]: invalid value ...`, isso significa
+que a string codificada com base64 nos dados foi decodificada com sucesso, mas não pôde ser analisada como um arquivo `.docker/config.json`.
+
+## Crie um Segredo fornecendo credenciais pela linha de comando
+
+Crie este Segredo, nomeando a `regcred`:
+
+```shell
+kubectl create secret docker-registry regcred --docker-server=<your-registry-server> --docker-username=<your-name> --docker-password=<your-pword> --docker-email=<your-email>
+```
+
+Onde:
+
+* `<your-registry-server>` é o seu `Private Docker Registry FQDN`.
+  Use `https://index.docker.io/v1/` para DockerHub.
+* `<your-name>` é o seu nome de usuário do Docker.
+* `<your-pword>` é a sua senha do Docker.
+* `<your-email>` é o seu email do Docker.
+
+Você definiu com sucesso as suas credenciais Docker no cluster como um Segredo chamado `regcred`.
+
+{{< note >}}
+Digitando segredos na linha de comando, pode armazená-los no histórico do seu shell, desprotegidos, e
+esses segredos também podem ser visíveis para outros usuários em seu PC durante o tempo que `kubectl` estiver em execução.
```

Comment:
Few issues with this block:
* Perhaps "Digitar" in lieu of "Digitando"?
* Extra comma after "comando".
* It would also be better to replace "segredos" here with "credenciais", as the context here is broader.